config/docker/security/rotate-secrets.sh

```
@@ -0,0 +1,130 @@
+    local file="$1"
+    local content="$2"
+    echo -n "$content" > "$file"
+}
+
+# Function to rotate database password
+rotate_db_password() {
+    local new_password=$(generate_random_string 32)
+    save_secret "$SECRETS_DIR/db_password.txt" "$new_password"
+    
+    # Update Vault if available
+    if [ -n "$VAULT_TOKEN" ] && [ -n "$VAULT_ADDR" ]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)